utils/types/requests.ts

```diff
@@ -0,0 +1,65 @@
+import { Attribute, Category, InventoryItem, ItemDefinition, User } from '.'
+
+export type UserPostRequest = User
+export interface UserPutRequest extends User {
+  _id: string
+}
+export type UserRequest = User
```

Comment:
@andlrutt what is the point of these base `request` objects?

Reply:
if PUT and POST have the some of the same changes to the base object, having the middle layer just saves on what needs to change in the derived post/put objects.